HW9__JDBC/src/main/java/CarDaoDelete.java

```diff
@@ -0,0 +1,58 @@
+import java.sql.*;
+
+public class CarDaoDelete {
```

Comment:
You should have `CarDao` class with all operations, not separate for each action.

Reply:
OK.